### DISCUSSION TITLE
Can’t Set WFM Bandwidth Lower Than 50 KHz

### DISCUSSION THREAD

**Discussion by AlexLandherr** [AlexandreRouma/SDRPlusPlus#1274] [Q&A] [answered]
I'm having some trouble setting the WFM bandwidth to anything lower than 50 KHz on my HackRF One r9.

Can anyone help me please? I'm on Debian 12 Bookworm.

**Comment by AlexandreRouma** [comment, marked as answer]
Assuming this is again related to APT, You're not supposed to use WFM... Use NFM at 44KHz with noise reduction enabled and set to the NOAA-APT preset.
